### ISSUE TITLE
NFC hi5s

### ISSUE THREAD

**Issue by nicklaslof** [Slashat/slashat-android#28]
Kanske till nästa meetup så de med NFC i telefonen kan hi5a med det istället.

Kräver dock att båda två har det men det är väl bara att öppna NFC med rätt info vid hi5-tillfället (samtidigt som man visar QR-kod osv) så väljer den var den får infon först

**Comment by tommienu** [comment]
Nein, så länge iPhone inte stödjer NFC så är det inget alternativ. Appen
måste fungera likadant över plattformarna.

2013/11/7 Nicklas Löf notifications@github.com

**Comment by nicklaslof** [comment]
precis. därför jag skrev att den kollar/visar både NFC och QR-kod samtidigt..
